src/main/java/ru/emdev/security/auth/util/PreferencesUtil.java

```diff
@@ -0,0 +1,128 @@
+package ru.emdev.security.auth.util;
+
+import java.util.Date;
+
+import com.liferay.portal.kernel.exception.SystemException;
+import com.liferay.portal.kernel.util.PrefsPropsUtil;
+import com.liferay.portal.kernel.util.StringPool;
+
+/**
+ * 
+ * @author Jose Campos
+ *
+ */
+public class PreferencesUtil {
+
+	public static int getMaxCountByCompany(long companyId) {
+
+		return getInteger(companyId, PropsKeys.MAX_SESSION_COUNT_FOR_USER, 0);
+
+	}
+
+	public static boolean isEnabledByCompany(long companyId) {
+
+		return getBoolean(companyId,
+				PropsKeys.EXTENDED_AUTH_ENABLED_BY_COMPANY, false);
+	}
+
+	public static boolean isEnabledOverrideUserSettings(long companyId) {
+
+		return getBoolean(companyId,
+				PropsKeys.EXTENDED_AUTH_ENABLED_OVERRIDE_USER_SETTINGS, false);
+	}
+
+	public static boolean isEnabledAccessAdmin(long companyId, long userId) {
```

Comment:
Do I understand this feature correctly: if this property set into true - then extended checking is avoided for admin users?